test/development/app-dir/missing-required-html-tags/index.test.ts

```diff
@@ -61,7 +61,18 @@ describe('app-dir - missing required html tags', () => {
       )
     )
 
-    // TODO(NDX-768): Should show "missing tags" error
-    await assertNoRedbox(browser)
+    if (process.env.TURBOPACK) {
+      await assertHasRedbox(browser)
+      // Wait for the HMR to apply and the updated error to show.
+      await retry(async () => {
+        expect(await getRedboxDescription(browser)).toEqual(outdent`
+          The following tags are missing in the Root Layout: <html>, <body>.
+          Read more at https://nextjs.org/docs/messages/missing-root-layout-tags
+        `)
+      })
+    } else {
+      // TODO(NDX-768): Should show "missing tags" error
+      await assertNoRedbox(browser)
+    }
```

Comment:
@eps1lon This is what should be happening, right?
I don't exactly understand why it's working now in Turbopack, but I'll take it.

Reply:
Seems like we're now throwing away the React root. That's why we're getting the full error. It didn't used to error because we just call `root.render(element)` which no longer errors when the container is the `document` and `element` no longer contains `<html>`

Can you double check that React/UI state is preserved after HMR updates in "normal" cases (e.g. no error and we just change some text)?

Reply:
So you're saying it's impossible for this to work correctly? It should still fail?

Reply:
It's still not working properly which isn't unexpected. It's just this specific case that now works. Fine to update this assertion for now.